MovieAppPetProject/MovieAppPetProject/Watchlist/WatchlistViewModel.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  WatchlistViewModel.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/05/14.
+//
+
+import Foundation
+
+class WatchlistViewModel {
+
+    // MARK: - Variable
+    private let coreDataManager: CoreDataManager
```

Comment:
same as earlier comment. ViewModel should not know about core data manager

Reply:
understood